app/routes/questionnaire.py

```diff
@@ -37,6 +37,10 @@
 @questionnaire_blueprint.before_request
 def before_questionnaire_request():
     metadata = get_metadata(current_user)
+
+    if cookie_session.get("submitted", False):
+        raise PostSubmissionException(401)
```

Comment:
Do we need a new exceptioon?. I think we should only raise a new one if we want to potentially log and alert based on that. This could be done similar to Rhys' change where the check is done in the error handler. This could raise `NoQuestionnaireStateException` which Rhys introduced. Then in the error handler we would do:
```py
if cookie_session.get("submitted", False):
    return _render_error_page(401, "submission-complete")
```
https://github.com/ONSdigital/eq-questionnaire-runner/pull/265/files#diff-0dd9f13aea9445b78356c3213cb91ab0R48-R56

So these 2 PRs are slightly inconsistent between each other right now.
Thoughts?

Reply:
There's no need to have multiple exceptions if one covers all the requirements, especially as it is just empty. Given that's the case, I'd just say I make those changes after Rhys's merges.